app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class UsersController < API::V1::ApplicationController
```

Comment:
I think it should work without the `API::V1::` namespace
```suggestion
    class UsersController < ApplicationController
```

Reply:
Yes, true!
Tested here:
![image](https://user-images.githubusercontent.com/77609814/122876525-cf683900-d35f-11eb-969a-e9e2e1ebec4c.png)

Thanks! :)